examples/widgets/popover_menu.nim

```diff
@@ -21,20 +21,32 @@
 # SOFTWARE.
 
 import owlkettle
+import owlkettle/[adw, playground]
 
 viewable App:
-  discard
+  sensitive: bool = true
+  sizeRequest: tuple[x, y: int] = (-1, -1) 
+  offset: tuple[x, y: int] = (0, 0)
+  position: PopoverPosition = PopoverBottom
 
 method view(app: AppState): Widget =
   result = gui:
     Window:
-      title = "Popover Menu Demo"
-      defaultSize = (300, 200)
-      
+      defaultSize = (800, 600)
+
       HeaderBar {.addTitlebar.}:
+        insert(app.toAutoFormMenu(ignoreFields = @["pixbuf", "loading"], sizeRequest = (400, 300))) {.addRight.}
```

Comment:
ignoreFields is not necessary, is sizeRequest necessary?

Reply:
I'd like to keep the sizeRequest as the default size of the menu is more suited to an amount of fields like Scale has.
Thus (in my opinion) it looks nicer if the height is just enough to cover the fields.
With the default values it looks like this:
![image](https://github.com/can-lehmann/owlkettle/assets/65814317/51a5953a-a47e-4dd8-82cd-a03ec13067de)